providers/shared/components/backupstore/id.ftl

```diff
@@ -0,0 +1,217 @@
+[#ftl]
+
+[@addComponent
+    type=BACKUPSTORE_COMPONENT_TYPE
+    properties=
+        [
+            {
+                "Type"  : "Description",
+                "Value" : "Store for holding backups"
+            }
+        ]
+    attributes=
+        [
+            {
+                "Names" : "Links",
+                "Description" : "Link to the key to be used for encrpytion of snapshots. If not provided, the baseline key will be used",
```

Comment:
I think if you want to have a specific function for a link it might be better as a dedicated attribute?

Reply:
When I checked our key support, it is currently only on the baseline component so I switched to just being able to say which key is required.